tests/fixtures/addon/npm/package.json

```diff
@@ -14,11 +14,13 @@
   },
   "scripts": {
     "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
     "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
     "start": "ember serve",
-    "test": "ember test",
-    "test:all": "ember try:each"
+    "test": "npm-run-all lint:* test:*",
+    "test:ember": "ember test",
+    "test:ember-try": "ember try:each"
```

Comment:
that means yarn test will now run ember try too? we explicitly changed it a while ago to not do that because running it takes forever and canceling it leaves your repo in a somewhat weird state.

Reply:
We should fix the cancellation issues (regardless of choice here), but IMHO this is a feature not a bug. All too many times I’ve been bitten by addons that are not using the default .travis.yml setup (using something like circle, GH actions, or other node aware tooling) that assume `npm test` means “good to go”.